src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -131,7 +137,10 @@ public bool IsEmpty(DocumentId documentId)
 
                 if (!await TryDeserializeDocumentAsync(serializer, document, builder, cancellationToken).ConfigureAwait(false))
                 {
-                    Contract.Requires(false, "How this can happen?");
+                    Contract.Requires(lastResult.Version == VersionStamp.Default);
```

Comment:
I presume this fixed https://github.com/dotnet/roslyn/issues/10904? If so, you may want to unskip the new FixAll VSI test that I am adding for v2 engine: https://github.com/dotnet/roslyn-internal/pull/919